windowsforms/Conway's-Game-of-Life/VB/Conway's-Game-of-Life/Conway's-Game-of-Life.vbproj

```diff
@@ -0,0 +1,58 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <RootNamespace>Sudoku</RootNamespace>
+    <StartupObject>Conways_Game_of_Life.Form1</StartupObject>
+    <RootNamespace>Conways_Game_of_Life</RootNamespace>
+    <AssemblyName>Conways Game of Life</AssemblyName>
+    <UseWindowsForms>true</UseWindowsForms>
+    <GenerateResourceUsePreserializedResources>true</GenerateResourceUsePreserializedResources>
+    <ApplicationIcon>Program\gui\conway_s_game_of_life.ico</ApplicationIcon>
+    <OptionStrict>On</OptionStrict>
+    <OptionInfer>Off</OptionInfer>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <NoWarn />
+    <WarningsAsErrors>41999,42016,42017,42018,42019,42020,42021,42022,42032,42036</WarningsAsErrors>
```

Comment:
Do you know what these errors are? 

In the default Framework template it looks like approximately the same group is set to NoWarn, so I'm curious about changing that. I'm also trying to figure out what these are :)

Reply:
Maybe TryConvert, I didn't. Deleting them has no effect on this project.